### ISSUE TITLE
Project fails at loading scripting assemblies

### ISSUE THREAD

**Issue by seifmostafa73** [ManakhovN/FigmaToUnityImporter#24]
after I tried to add a page from my Figma profile to unity the projects froze and when trying to reopen it gets stuck at loading the assembly, tried to remove and rebuild the packages and project settings folders but it still stuck.
Note: I tried to add a page before using an older version and it was working fine, this only happened after updating to the latest version.

**Comment by ManakhovN** [comment]
@seifmostafa73 Could you please write the exact steps of what have u done to achieve that result (from adding the tool till the freeze). Because everything works in my test project. And maybe you can share the page that froze the project or the project itself.

**Comment by seifmostafa73** [comment]
yeah sure, I basically had a new OS [linux] and had a fresh setup of unity and then just copied the Github link to add the package referred to in the install section "https://github.com/ManakhovN/FigmaToUnityImporter.git?path=/Assets/FigmaImporter to Package Manager".
next, I just went over and copied the authentication link and continued as the Usage section stated.
as for the exact page, it was a modified version of this project's Board page 
https://www.figma.com/community/file/888906989876942858
for the delayed reply, Github doesn't push notifications.
